src/hooks/useWalletConnectEventsManager.ts

```diff
@@ -147,22 +147,22 @@ export default function useWalletConnectEventsManager(initialized: boolean) {
 					}
 				case EIP155_SIGNING_METHODS.ETH_SEND_TRANSACTION:
 				case EIP155_SIGNING_METHODS.ETH_SIGN_TRANSACTION:
-					const ethTX = request.params[0]
+					const ethTx = request.params[0]
 					const accountTX = getAccountWithAddressAndChainId(
 						accounts,
-						ethTX.from,
+						ethTx.from,
 						chainId,
 					)
 					if (!!accountTX) {
 						try {
 							const walletApiClient = initWalletApiClient()
-							const liveTx = convertEthToLiveTX(ethTX)
 							addPendingFlow({
 								id,
 								topic,
 								accountId: accountTX.id,
```

Comment:
Could we add some logic to add the type of action to being able to check if we don't send an empty data when it's a swap?
For example we could have:
```
type PendingFlow = {
        type: 'send' | 'swap'
	id: number
	topic: string
	accountId: string
	message?: string
	isHex?: boolean
	ethTx?: EthTransaction
}
```

Then in triggerPendingFlow we will know if it was a swap and we can make sure we don't call signAndBroadcast without data, wdyt?
It's not perfect but at least whatever happened inside zustand we will not send a partial swap request.